### PR TITLE
Javascript loops practice

### PR DESCRIPTION
**Pull request by Jc-Leon**
Screen Shot 2021-07-09 at 1 52 30 AM
Screen Shot 2021-07-09 at 1 52 22 AM
hey cody i went to after hours like you said and it actually helped me guide me in the right direction but just ran into 1 issue if you could tell me what tell me what i did wrong i'd appreciate it 🥺

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude! Your problem is that your function is treating `"7"` as a seven, but that's a string, so it should not be considered as the number `7`

**Comment by Jc-Leon** [comment]
i understand what youre telling me but how come on the other tests it allowed them to go through but on this one it didnt or is it  something im missing?

**Comment by Jc-Leon** [comment]
Screen Shot 2021-08-02 at 7 49 55 PM

File: javascript-loops-practice/includes-seven.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /* exported includesSeven */
 function includesSeven(array) {
   for (var i = 0; i < array.length; i++) {
-    if (array.length <= 7) {
+    if (Number.isInteger(array[i]) && array[i] <= 7) {
```
Comment (codingcodymiller) [inline javascript-loops-practice/includes-seven.js:4]: This is checking for numbers that are less than or equal to 7, the function is meant to check for the presence of the `number` 7 only

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Pull request merged**